<launch>

	<arg name="state_inspection" default="8" />
	<arg name="parking_state" default="-2" />

	<node name="waypoint_loader_node" pkg="waypoint_maker" type="waypoint_loader" output="screen">
		<param name="state_inspection" value="$(arg state_inspection)" />
		<param name="parking_state" value="$(arg parking_state)" />
	</node>

</launch>
